### ISSUE TITLE
Allow to filter comments in a bug according to username

### ISSUE THREAD

**Issue by vaibhavmagarwal** [AutomatedTester/Bugsy#11]
I am using bugsy to get comments of some bugs. I referred to this: http://bugsy.readthedocs.org/en/latest/comment.html , however I cannot see an option to filter the comments of a bug according to username. For example, "filter all comments of user tbplbot@gmail.com (TBPL Robot) in bug 1081702"

**Comment by vaibhavmagarwal** [comment]
I cannot figure out from the docs to do the above issue, I am sorry if there is already a way to do it.... Please let me know if there is.

**Comment by AutomatedTester** [comment]
I haven't added properties for everything but I think I should just for the
value of getting better docs...

I think the following

```
  tbpl_comments =  [comment for comment in bug.get_comments() if
comment.creator == 'tbpl robot']
```

would work.

David Burns
Email: david.burns@theautomatedtester.co.uk
URL: http://www.theautomatedtester.co.uk/

**Comment by vaibhavmagarwal** [comment]
Awesome! This worked for me. @AutomatedTester should I send in a pull request to update the documentation for get_comments section?

**Comment by AutomatedTester** [comment]
Nah, I am going to use this bug to make sure that I add the properties to
comments with documentation.

David Burns
Email: david.burns@theautomatedtester.co.uk
URL: http://www.theautomatedtester.co.uk/

**Comment by vaibhavmagarwal** [comment]
Cool! Thanks again for the help.

**Comment by parkouss** [comment]
I suspect that this issue could be closed now because the documentation is updated: http://bugsy.readthedocs.org/en/latest/comment.html#bugsy.Comment

**Comment by AutomatedTester** [comment]
Landed in 8865dfc3f3a6e8ff6425c54149eb7c2663b46e69
